cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/etl/batch/ETLWorkflowTestRun.java

```diff
@@ -217,6 +220,35 @@ public void testBackwardsCompatibleExternalDatasetTrackingSpark() throws Excepti
     testExternalDatasetTracking(Engine.SPARK, true);
   }
 
+  @Test
+  public void testGetPlugins() throws Exception {
```

Comment:
this test does not belong here, as its not testing anything about hydrator. It belongs in some app lifecycle handler test.